pkg/ts/isolation/ses.ts

```diff
@@ -23,16 +23,27 @@ export const initSes = (options?) => {
   if (!particleCompartment) {
     const debugOptions = {
       consoleTaming: 'safe',
-      errorTaming: 'unsafe',
-      errorTrapping: 'unsafe',
+      errorTaming: 'safe',
+      errorTrapping: 'safe',
       stackFiltering: 'concise'
     };
     const prodOptions = {};
-    requiredLog.log('LOCKDOWN');
-    requiredLog.groupCollapsed('...removing intrinics...');
-    lockdown(debugOptions || prodOptions);
-    requiredLog.groupEnd();
-    particleCompartment = new Compartment({log, resolve, html, makeKey, timeout, ...options?.injections, harden: globalThis.harden});
+    requiredLog.groupCollapsed('LOCKDOWN');
+    try {
+      lockdown(debugOptions || prodOptions);
+      const utils = {log, resolve, html, makeKey, timeout};
+      particleCompartment = new Compartment({
```

Comment:
This looks like it's more than just logging (which is what PR description says). Maybe this fixes the hardening issue @cromwellian found (just seeing line 41 below)?

Asking:
1. So you can move it out of this PR if you really did intend it to be logging specific
2. So you can consider editing PR/commit description if you feel warranted

Reply:
...re-reading more closely (line wrap) maybe this is pure formatting change, my bad.

Reply:
Good looking out. I'm still in the habit of just polishing stuff as I go, which leads to messy PRs. I'll need to be more disciplined.